templates/repo/graph.tmpl

```diff
@@ -6,57 +6,44 @@
 			<h2 class="ui header dividing">{{.i18n.Tr "repo.commit_graph"}}
 				<div class="ui right">
 					<div class="ui icon buttons tiny color-buttons">
+						<div class="ui multiple selection search dropdown" id="flow-select-refs-dropdown">
+							<input type="hidden">
+              				<i class="dropdown icon"></i>
+              				<input type="text" class="search">
+              				<div class="default text">{{.i18n.Tr "repo.commit_graph.select"}}</div>
```

Comment:
```suggestion
						<i class="dropdown icon"></i>
						<input type="text" class="search">
						<div class="default text">{{.i18n.Tr "repo.commit_graph.select"}}</div>
```

Reply:
@silverwind whitespace issue?

Reply:
Yes, there's a bunch of spaces before the tabs.

Reply:
We certainly need a linter rule that forbids space indentation on `.tmpl` and `.go` files, not sure which tool to use, thought.